src/modules/site/templates/data/vbrowser.html

```diff
@@ -491,45 +479,46 @@ <h3 class="text-center">Variant Impact</h3>
   let result = '<mark></mark>';
   let strains = data.split(',');
   let i = 0;
-  for(s of strains) {
+  for(strain of strains) {
     if ((++i % strains_per_row) === 0) {
       result += '<br/>'
     }
-    result += `<mark class="mark_${s}">${s},</mark>`;
+    result += `<mark class="mark_${strain}">${strain},</mark>`;
   }
   return result;
 }
 
 
 function init_data_table() {
+  const colDefs = columnList.map((col, idx) => {
+    switch(col.id) {
+      case 'strains':
+        return { 
+          "name": col.id, 
+          "mRender": renderStrainsCol,
+          "targets": idx
+        } 
+      case 'target_consequence':
+        return { 
+          "name": col.id, 
+          "mRender": renderTargetConsequenceCol,
+          "targets": idx
+        } 
+      default:
+        return { 
+          "name": col.id, 
+          "targets": idx

```

Comment:
can you keep the shape of the returned value? ie. assign the `mRender` to a no-op-function like `nop = () => ''`.

Reply:
It looks like the mRender value (now called render in datatables 1.10) has to be null or undefined. It'll consider it valid if it's type function and will try to execute.

I refactored to retain the shape of the return value and just get the render value based on the col.id